AvalonStudio/AvalonStudio/ShellViewModel.cs

```diff
@@ -366,8 +366,6 @@ public IToolBar ToolBar
 
         public CancellationTokenSource ProcessCancellationToken { get; private set; }
 
-        public IEnumerable<IEditorProvider> EditorProviders => _editorProviders;
```

Comment:
You need to make same change from MinimalShell.cs (we have some command line tools which need to import certain types of non-ui based extensions) That probably why the build is failing. At somepoint we can probably remove the dependency in the command line tool (avalon build) project on IShell to prevent this problem.